src/main.rs

```diff
@@ -183,6 +192,23 @@ pub struct Log {
     pub message: Option<String>,
 }
 
+fn serde_json_simple_test() {
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
In the list of things to derive, throw in `PartialEq`. Then `assert_eq!` will work.